ExpoKit.podspec

```diff
@@ -19,21 +19,12 @@ Pod::Spec.new do |s|
     ss.preserve_paths = "ios/Exponent/**/*.{h,m}"
     ss.exclude_files = "ios/Exponent/Supporting/**", "ios/Exponent/Versioned/Optional/**/*.{h,m}"
 
-    ss.dependency 'Amplitude-iOS', '~> 4.6'
```

Comment:
Amplitude should be part of the Expo Client dependencies (though not ExpoKit), as it's used in EXAnalytics

Reply:
Ah, yeah, I haven't noticed this 🙈 thanks for pointing this out!